tools/install-powershell.ps1

```diff
@@ -79,6 +79,68 @@ Function Remove-Destination([string] $Destination) {
     }
 }
 
+Function IsPathInRegistryPath([string] $Path) {
```

Comment:
should be named `Test-PathInRegistryPath`

Reply:
done